worlds/celeste64/Options.py

```diff
@@ -73,6 +75,93 @@ class Carsanity(Toggle):
     """
     display_name = "Carsanity"
 
+class Checkpointsanity(Toggle):
+    """
+    Whether activating Checkpoints grants location checks
+
+    Activating this will also shuffle items into the pool which allow usage and warping to each Checkpoint
+    """
+    display_name = "Checkpointsanity"
+
+
+class ColorChoice(TextChoice):
+    option_strawberry = 0xDB2C00
+    option_empty = 0x6EC0FF
+    option_double = 0xFA91FF
+    option_golden = 0xF2D450
+    option_baddy = 0x9B3FB5
+    option_fire_red = 0xFF0000
+    option_maroon = 0x800000
+    option_salmon = 0xFF3A65
+    option_orange = 0xD86E0A
+    option_lime_green = 0x8DF920
+    option_bright_green = 0x0DAF05
+    option_forest_green = 0x132818
+    option_royal_blue = 0x0036BF
+    option_brown = 0xB78726
+    option_black = 0x000000
+    option_white = 0xFFFFFF
+    option_grey = 0x808080
+    option_any_color = -1
+
+    @classmethod
+    def from_text(cls, text: str) -> Choice:
+        text = text.lower()
+        if text == "random":
+            choice_list = list(cls.name_lookup)
+            choice_list.remove(-1)
```

Comment:
```suggestion
            choice_list.remove(cls.option_any_color)
```
Absolutely a nitpick, but would allow you to adjust the underlying value if necessary, say if a "rainbow" were to be added as -1 and this was moved to -2.